src/components/solverComponents/InputButton.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { MouseEventHandler, ReactElement } from "react";
+import { Tooltip, Button } from "@chakra-ui/react";
+
+
+export const InputButton = (props: { icon: ReactElement, text: string, toolTipText: string, onClick?: MouseEventHandler<HTMLButtonElement> | undefined }) => (
+    <Tooltip label={props.toolTipText} color='white'>
+        <Button colorScheme='teal' onClick={props.onClick} rightIcon={props.icon}
+                width="200px" size="lg" >
```

Comment:
no fixed width, please! if you want extra space at the side, use [margin or padding props](https://chakra-ui.com/docs/styled-system/style-props) from chakra ui. If you want to align the component with another one, use a layout like Grid.

Reply:
is now a grid